{"metadata":{"usedHelpers":[],"marked":[],"modules":{"imports":[],"exports":{"exported":[],"specifiers":[]}}},"options":{"filename":"packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js","filenameRelative":"packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js","env":{"development":{"plugins":[]}},"retainLines":false,"highlightCode":true,"suppressDeprecationMessages":false,"presets":[],"plugins":[[[],{"generateLetDeclarations":true,"enforceStrictMode":false}],[[],null],[[],{"polyfill":false}],[[],null],[[],null],[[],null],[[],{"allowTopLevelThis":true,"strict":false,"loose":true}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],{"loose":true}],[[],{"loose":true}],[[],null],[[],{"loose":true}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],{"loose":true}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null]],"ignore":[],"code":true,"metadata":true,"ast":true,"comments":true,"compact":false,"minified":false,"sourceMap":true,"sourceMaps":true,"sourceMapTarget":"packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js.map","sourceFileName":"packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js","babelrc":false,"sourceType":"module","moduleIds":false,"passPerPreset":false,"parserOpts":false,"generatorOpts":false,"basename":"after.hook.error"},"ignored":false,"code":"describe('spec 1', function () {\n  after(function () {\n    console.log('after');\n    throw new Error('after hook error');\n  });\n  it('should be called because error is in after hook', function () {\n    console.log('test 1');\n  });\n  it('should be called because error is in after hook', function () {\n    console.log('test 2');\n  });\n});\ndescribe('spec 2', function () {\n  it('should be called, because hook error was in a sibling suite', function () {\n    console.log('test 3');\n  });\n});","map":{"version":3,"sources":["packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js"],"names":["describe","after","console","log","Error","it"],"mappings":"AAAAA,SAAS,QAAT,EAAmB,YAAY;AAC7BC,QAAM,YAAY;AAChBC,YAAQC,GAAR,CAAY,OAAZ;AACA,UAAM,IAAIC,KAAJ,CAAU,kBAAV,CAAN;AACD,GAHD;AAIAC,KAAG,iDAAH,EAAsD,YAAY;AAChEH,YAAQC,GAAR,CAAY,QAAZ;AACD,GAFD;AAGAE,KAAG,iDAAH,EAAsD,YAAY;AAChEH,YAAQC,GAAR,CAAY,QAAZ;AACD,GAFD;AAGD,CAXD;AAYAH,SAAS,QAAT,EAAmB,YAAY;AAC7BK,KAAG,6DAAH,EAAkE,YAAY;AAC5EH,YAAQC,GAAR,CAAY,QAAZ;AACD,GAFD;AAGD,CAJD","file":"packages/practicalmeteor:mocha/test/integration/fixtures/hooks/after.hook.error.js.map","sourcesContent":["describe('spec 1', function () {\n  after(function () {\n    console.log('after');\n    throw new Error('after hook error');\n  });\n  it('should be called because error is in after hook', function () {\n    console.log('test 1');\n  });\n  it('should be called because error is in after hook', function () {\n    console.log('test 2');\n  });\n});\ndescribe('spec 2', function () {\n  it('should be called, because hook error was in a sibling suite', function () {\n    console.log('test 3');\n  });\n});\n"]},"hash":"b4875b4d18532b35db815a855659be27f2743d4d"}
